NBitcoin/Crypto/SchnorrSignature.cs

```diff
@@ -0,0 +1,86 @@
+using NBitcoin.BouncyCastle.Asn1;
+using NBitcoin.BouncyCastle.Math;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NBitcoin.BouncyCastle.Asn1.X9;
+using System.Security.Cryptography;
+
+namespace NBitcoin.Crypto
+{
+	public class SchnorrSigner
+	{
+
+        public byte[] Sign(uint256 m, byte[] secret)
```

Comment:
`secret` should be of type `uint256`.

Reply:
There are two overloaded `Sign` methods, one receives a `Key` instance and the other one receives a `BigInteger` instance.